ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/priority_flow_control.schema.yml

```diff
@@ -0,0 +1,59 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  priority_flow_control:
+    documentation_options:
+      filename: data_model/Quality of Service
+    type: dict
+    description: |
+      Global Priority Flow Control settings.
+    keys:
+      all_off:
+        type: bool
+        description: |
+          Disable PFC on all interfaces.
+      watchdog:
+        type: dict
+        keys:
+          action:
+            type: str
+            valid_values: ["drop", "no-drop"]
+            description: |
+              Action on stuck queue. Default is 'errdisable'.
+              Aceptable values are "drop" or "no-drop".
+          timeout:
+            type: str
+            pattern: "\\d+(\\.\\d)?"
+            convert_types:
+              - "int"
+            description: |
+              Timeout after which port should be errdisabled or
+              should start dropping on congested priorities.
+              This should be decimal with up to 1 decimal point.
+              Example : 0.5 or 60
+          polling_interval:
+            type: str
+            pattern: "\\d+(\\.\\d)?"
```

Comment:
The pattern here would break with `auto`. We have not really started using `pattern` anywhere else, so I suggest you just remove it for now from all the vars here.

Reply:
BTW also add the unit to all your descriptions. So "time inverval in seconds ..."

Reply:
Agreed on both points. Updated the schema.